src/twisted/application/runner/_pidfile.py

```diff
@@ -91,26 +96,23 @@ class PIDFile:
     _log = Logger()
 
     @staticmethod
-    def _format(pid):
+    def _format(pid: int) -> bytes:
         """
         Format a PID file's content.
 
         @param pid: A process ID.
-        @type pid: int
```

Comment:
Is it ok to remove pydoctor type annotation now?

I see in pydoctor that `Type annotations on function definitions are not supported yet`

Reply:
They are supported in the `master` branch, but not yet in a release. I'll try to get a pydoctor release out soon.